Put public/private keys in DATADIR/keys (aka /usr/share/asterisk/keys) instead
of VARLIBDIR/keys. Besided being FHS compliant, that's where the Makefile
installs them and what src/main/asterisk.c does when parsing asterisk.conf.

Forwarded as #10811.

 -- Faidon Liambotis <paravoid@debian.org>

--- asterisk-1.4.9~dfsg.orig/build_tools/make_defaults_h
+++ asterisk-1.4.9~dfsg/build_tools/make_defaults_h
@@ -14,7 +14,7 @@ cat << END
 #define AST_DATA_DIR   "${INSTALL_PATH}${ASTDATADIR}"
 #define AST_LOG_DIR    "${INSTALL_PATH}${ASTLOGDIR}"
 #define AST_AGI_DIR    "${INSTALL_PATH}${AGI_DIR}"
-#define AST_KEY_DIR    "${INSTALL_PATH}${ASTVARLIBDIR}/keys"
+#define AST_KEY_DIR    "${INSTALL_PATH}${ASTDATADIR}/keys"
 #define AST_DB         "${INSTALL_PATH}${ASTVARLIBDIR}/astdb"
 #define AST_TMP_DIR    "${INSTALL_PATH}${ASTSPOOLDIR}/tmp"
 
